addons/delivery/wizard/choose_delivery_package.py

```diff
@@ -55,8 +41,14 @@ def _onchange_packaging_weight(self):
             return {'warning': warning_mess}
 
     def put_in_pack(self):
-        # write shipping weight and product_packaging on 'stock_quant_package' if needed
-        if self.delivery_packaging_id:
-            self.stock_quant_package_id.packaging_id = self.delivery_packaging_id
-            if self.shipping_weight:
-                self.stock_quant_package_id.shipping_weight = self.shipping_weight
+        picking_id = self.env.context.get('picking_id')
+        if picking_id:
+            picking = self.env['stock.picking'].browse(picking_id)
+            move_line_ids = picking.move_line_ids.filtered(lambda ml: ml.qty_done > 0 and not ml.result_package_id)
+            self.stock_quant_package_id = picking._put_in_pack(move_line_ids)
+            delivery_package = self.stock_quant_package_id
```

Comment:
Remove stock_quant_package_id from fields instead only use a variable since it's only used in this method